### ISSUE TITLE
Outline doesn't work with operator ...

### ISSUE THREAD

**Issue by Hraesvelg** [Hirse/brackets-outline-list#103]
hi,

i use brackets-outline-list since the beginning but i have a problem with a special operator of es6. the outline is break when i use the operator of "decomposition". Outline show a message of syntaxe error...

* **Brackets Version: 1.11.0** 
* **Brackets Outline List Version: 1.4.0** 

```js
let obj1 = {foo : bar, bar : foo};
let obj2 = {...obj1, toto : tata}; // ERROR
```
thanks !

**Comment by Hirse** [comment]
Fixed in b0ba3e1b577efdac8dc45bc63acc3fe67705e8ba, should be available in the next update.

**Comment by Hraesvelg** [comment]
thanks ! it's fixed with the last update !

good continuation !
